docs/source-fabric/conf.py

```diff
@@ -254,7 +254,23 @@
     "python": ("https://docs.python.org/3", None),
     "torch": ("https://pytorch.org/docs/stable/", None),
     "pytorch_lightning": ("https://lightning.ai/docs/pytorch/stable/", None),
+    "lightning": ("https://lightning.ai/docs/app/stable//", None),
```

Comment:
```suggestion
    "lightning": ("https://lightning.ai/docs/app/stable/", None),
```

Reply:
Why is this pointing to docs/app? Seems counter-intuitive. Why is intersphinx needed between lightning and lightning

Reply:
Because you are referring to docs `LightningApp` and `LightningWork` so shall we remove the reference?

Reply:
Better to remove, otherwise we get in trouble if we want to reference other things like `lightning.pytorch` etc.